dev-python/sphinxcontrib-programoutput/sphinxcontrib-programoutput-0.15.ebuild

```diff
@@ -0,0 +1,48 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
```

Comment:
Prefer the maintainers style when bumping their packages, ie no quotation marks here.